### ISSUE TITLE
movielens dataset

### ISSUE THREAD

**Issue by gsmeros** [sbyrnes/likely.js#10]
can this work with the movielens dataset? (userId,movieId,rating).
is this convertable into your inputmatrix userId - ratings ?

**Comment by sbyrnes** [comment]
Yes, there is no reason that would not work. If you do implement a transform that converts Movielens format into likely input matricies please do submit a tool to the project via PR so everyone can try it!

Likewise, if there are updates to the Likely input format that would make it easier let me know.
